kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/FileSystemViewKeyInputFormat.java

```diff
@@ -19,6 +19,7 @@
 import com.google.common.collect.Lists;
 import java.io.IOException;
 import java.util.Iterator;
+import java.util.LinkedList;
```

Comment:
Yep, I put some timings in this class and didn't get them cleanly removed.  I'll revert this class changes.